packages/db/src/types/migration.ts

```diff
@@ -0,0 +1,24 @@
+export interface MigrationResult {
```

Comment:
I'm wondering if these more properly belong in the Core types package?     They're .... database migrations, but not technically *database* types.

Reply:
Hm I think this is definitely a good point. I'm inclined to leave it here because they are related to the migration even though they aren't technically database types

Reply:
Not a hill I'm willing to die on.        If it weren't two migration scripts and/or we were keeping these around I'd be far more inclined to make a fuss about pulling it back in to the Lambda, but none of that needs to hold up the PR.      I'll consider the offline convo enough to unblock.

Reply:
Okay, after the offline conversation, I have decided to move the types. I'm not sure if I should create a directory or if this is sufficient [`f81181c` (#2173)](https://github.com/nasa/cumulus/pull/2173/commits/f81181cae9208ce31628820c487010838e22dca3)